src/components/BlogCard/BlogCard.react.js

```diff
@@ -0,0 +1,107 @@
+//@flow
+
+import * as React from "react";
+import Card from "../Card/Card.react";
+import Icon from "../Icon/Icon.react";
+
+type Props = {|
+  +children?: React.Node,
+  +title?: string,
+  +titleUrl?: string,
+  +avatarUrl?: string,
+  +description?: string,
+  +date?: string,
+  +imgUrl?: string,
+  +iconName?: string,
+  +iconHref?: string,
+  +authorName?: string,
+  +avatarImgSrc?: string,
+  +imgSrc?: string,
+  +imgAlt?: string,
+  +aside?: boolean,
+  +postHref?: string,
+|};
+
+function BlogCard({
+  children,
+  title,
+  description,
+  avatarUrl,
+  imgUrl,
+  imgAlt,
+  aside,
+  authorName,
+  date,
+  imgSrc = " ",
+  avatarImgSrc = " ",
+  iconName,
+  iconHref,
+  postHref,
+}: Props): React.Node {
+  return !aside ? (
+    <Card>
+      <a href={postHref}>
+        <img className="card-img-top" src={imgSrc} alt={imgAlt} />
+      </a>
+      <Card.Body className="d-flex flex-column">
+        <h4>
+          {" "}
+          <a href={postHref}>{title}</a>
+        </h4>
+        <div className="text-muted">{description}</div>
+        <div className="d-flex align-items-center pt-5 mt-auto">
+          <div
+            className="avatar avatar-md mr-3"
+            style={{ backgroundImage: `url(${avatarImgSrc}` }}
+          />
+          <div>
+            <a href="./profile.html" className="text-default">
```

Comment:
The href for this should be a prop in my opinion, shouldn't always point to `./profile.html`